migrate/20200103163139_rename_comfy_cms_blocks_to_comfy_cms_fragments.rb

```diff
@@ -0,0 +1,36 @@
+class RenameComfyCmsBlocksToComfyCmsFragments < ActiveRecord::Migration[5.2]
```

Comment:
Got this error while trying to rollback this migration:

== 20200103163139 RenameComfyCmsBlocksToComfyCmsFragments: reverting ==========
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:




This migration uses change_column, which is not automatically reversible.
To make the migration reversible you can either:
1. Define #up and #down methods in place of the #change method.
2. Use the #reversible method to define reversible behavior.